libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/tripsession/TripSessionStarterViewModel.kt

```diff
@@ -0,0 +1,82 @@
+package com.mapbox.navigation.dropin.component.tripsession
+
+import android.annotation.SuppressLint
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.dropin.component.navigation.NavigationState
+import com.mapbox.navigation.dropin.component.navigation.NavigationStateViewModel
+import com.mapbox.navigation.dropin.lifecycle.UIViewModel
+import com.mapbox.navigation.utils.internal.logI
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.launch
+
+sealed class TripSessionStarterAction {
```

Comment:
Probably `TripSessionAction`

Reply:
Responded here https://github.com/mapbox/mapbox-navigation-android/pull/5594#discussion_r833745023

Let's keep a single thread for discussing name possibilities